lib/checkunusedvar.cpp

```diff
@@ -1623,15 +1623,16 @@ bool CheckUnusedVar::isFunctionWithoutSideEffects(const Function& func, const To
             if (bodyVariable->isGlobal() ||
                 (std::find(pointersToGlobals.begin(), pointersToGlobals.end(), bodyVariable) != pointersToGlobals.end()))
```

Comment:
Now we can use `pointersToGlobals.find(bodyVariable) != pointersToGlobals.end()` instead here.